src/indicatorBuilder.js

```diff
@@ -127,7 +127,9 @@ const init = (chart_series, indicator) => {
          before_add_callback && before_add_callback();
 
          //Add indicator for the main series
-         chart_series[0].addIndicator(state.id, options);
+         if (chart_series !== "undefined") {
+            chart_series[0].addIndicator(state.id, options);
+         }
```

Comment:
Can we making it simpler? 
```suggestion
            chart_series[0]?.addIndicator(state.id, options);
```

Reply:
I was not sure if optional chaining works as expected in older projects. I made the code simpler as requested.

Reply:
Yes, you are right! my bad, optional chaining is not working as expected in older project!